docs/reference/garbage-collection.md

```diff
@@ -394,3 +393,72 @@ spark-submit --class io.treeverse.clients.GCBackupAndRestore \
   objects-to-restore-list-location backup-external-location storage-namespace azure
   ```
 </div>
+
+## Beta: Deleting uncommitted objects
+
+Deletion of objects that were never committed was always a difficulty for lakeFS, see
+[#1933](https://github.com/treeverse/lakeFS/issues/1933) for more details. Examples for 
+objects that will be collected as part of the uncommitted GC job:
```

Comment:
```suggestion
Run this job to [delete objects that were never committed and have no valid pointers from lakeFS](https://github.com/treeverse/lakeFS/issues/1933). This may happen when:
```